src/main/resources/static/views/guestsoverview.html

```diff
@@ -5,6 +5,7 @@
     + add Guest
 </button>
 <button type="button" class="btn btn-default" onclick="getSelectGuest()">Edit</button>
+<button type="button" class="btn btn-default" onclick="removeSelectedGuest()">x Delete</button>
```

Comment:
volgens mij kun je hier ipv x ook &times; neerzetten (volgens mij is dat mooier)